schema/schema_test.go

```diff
@@ -0,0 +1,183 @@
+package schema_test
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/hpcloud/termui/schema"
+
+	term "github.com/cloudfoundry/cli/cf/terminal"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Should I be seeing Godep.json changes? We don't seem to be pulling in any version of this stuff.